dotnet-authserver/src/TeacherIdentity.AuthServer/Infrastructure/RateLimiting/FixedWindowOptions.cs

```diff
@@ -0,0 +1,7 @@
+namespace TeacherIdentity.AuthServer.Infrastructure.RateLimiting;
+
+public class FixedWindowOptions
+{
+    public TimeSpan Window { get; set; } = TimeSpan.Zero;
```

Comment:
Nit: is `TimeSpan.Zero` required here?

Could we mark both properties `required` or would that break something?

Reply:
done